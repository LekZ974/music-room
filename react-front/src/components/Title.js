import React from 'react';

import { Title as NBTitle } from 'native-base';

const Title = props => {
  return <NBTitle {...props} />;
};
export default Title;
