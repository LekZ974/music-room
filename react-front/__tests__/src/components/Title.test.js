import React from 'react';
import renderer from 'react-test-renderer';
import Title from '../../../src/components/Title';

it('renders correctly', () => {
  const tree = renderer.create(<Title>Snapshot test with Title Component!</Title>).toJSON();

  expect(tree).toMatchSnapshot();
});
