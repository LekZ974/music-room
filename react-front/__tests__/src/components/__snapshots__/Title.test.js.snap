// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  numberOfLines={1}
  style={
    Object {
      "color": "#000",
      "fontFamily": "System",
      "fontSize": 17,
      "fontWeight": "700",
      "marginLeft": undefined,
      "paddingLeft": 4,
      "paddingTop": 1,
      "textAlign": "center",
    }
  }
>
  Snapshot test with Title Component!
</Text>
`;
